lib/grammers-client/src/client/channel.rs

```diff
@@ -0,0 +1,75 @@
+use super::Client;
+use anyhow::{anyhow, Result};
+use grammers_tl_types::{
+    enums::{
+        channels::ChannelParticipants::{NotModified, Participants},
+        messages::ChatFull,
+        ChannelParticipantsFilter,
+        InputChannel::Channel,
+        User,
+    },
+    functions::channels::{GetFullChannel, GetParticipants},
+    types::InputChannel,
+};
```

Comment:
The library generally avoids this style of imports, so I think we should stick with flat `use`s for consistency.